doc/Integrate-iOS.md

```
@@ -31,7 +31,27 @@ After installation, you will need to integrate CleverTap SDK into your app.
   CleverTap.autoIntegrate() // integrate CleverTap SDK using the autoIntegrate option
   CleverTapPlugin.sharedInstance()?.applicationDidLaunch(options: launchOptions)
   ```
-  
+
+### Handling deep links under the UIScene lifecycle
+
+If your app uses the `UIScene` lifecycle (a `SceneDelegate` with a `UIApplicationSceneManifest` in `Info.plist`), the cold-launch push payload is delivered to `scene(_:willConnectTo:options:)` rather than `didFinishLaunchingWithOptions:`. In that case `launchOptions` is `nil`, so you must ALSO notify the plugin from your SceneDelegate for `getInitialUrl` to return the deep link on a cold launch.
+
+  ###### Objective-C
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h4; Actual: h6

(MD001, heading-increment)